controllers/grid/users/reviewer/form/AdvancedSearchReviewerForm.php

```diff
@@ -144,6 +144,16 @@ public function fetch($request, $template = null, $display = false)
         $warnOnAssignment = array_merge($warnOnAssignment, $userIds);
         $warnOnAssignment = array_values(array_unique(array_map('intval', $warnOnAssignment)));
 
+        $locale = Locale::getLocale();
+        $submissionAuthors = $this->getSubmission()->getCurrentPublication()->getData('authors');
+        $authorAffiliations = [];
+        $authors = [];
+        foreach($submissionAuthors as $submissionAuthor) {
+            $affiliation = $submissionAuthor->getData('affiliation')[$locale];
```

Comment:
Can you change this line to:
```php
$affiliation = $submissionAuthor->getLocalizedData('affiliation');
```
This is a general pattern to retrieve localized data in the current (request) locale.

Reply:
Thanks for that. Sorted